tests/unit/test_core_reqid.py

```
@@ -0,0 +1,127 @@
+import warnings
+
+import pytest
+
+from notebooklm._core import ClientCore
+from notebooklm.auth import AuthTokens
+
+
+def _make_core() -> ClientCore:
+    auth = AuthTokens(
+        cookies={"SID": "test"},
+        csrf_token="test_csrf",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "csrf_token"

(S106)